server/desktop/.gitignore

```diff
@@ -1 +1,2 @@
-/build
\ No newline at end of file
+/build
+config/
```

Comment:
This seems fine but is a very generic word and could have consequences from ignoring generically, can we use absolute path?

Reply:
ah sure